Tests/ExtensionsTests.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ExtensionsTests.swift
+//  FramezillaTests
+//
+//  Created by Стас Клюхин on 09/07/2019.
```

Comment:
Wrong headers. @artemnovichkov, could you help with default header for our example project?

Reply:
Sure, let's fix it!